examples/shardingsphere-sample/shardingsphere-example-generator/src/main/resources/template/pom.ftl

```diff
@@ -16,22 +16,44 @@
   ~ limitations under the License.
   -->
 
+<#assign package="" />
+<#if feature?split(",")?size gt 1>
+    <#assign package="mixed" />
+<#else>
+    <#assign package = feature?replace('-', '.') />
+</#if>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.apache.shardingsphere.example</groupId>
         <artifactId>shardingsphere-jdbc-sample</artifactId>
-        <version>${revision}</version>
+        <version>${r'${revision}'}</version>
```

Comment:
is it correct with prefix `r`?

Reply:
It‘s correct, in order to output `${}` as is.